CONTRIBUTING.md

```diff
@@ -229,8 +164,8 @@ This is the slowest part of package updates. If any packages are skipped during
 1. After each package version update in [`package.json`](./package.json) you'll run the follow scripts
    - `$ yarn test`, if it fails you'll need to run `$ yarn test:dev` and update the related tests
    - `$ yarn build`, if it fails you'll need to run `$ yarn test:integration-dev` and update the related tests
-   - Make sure Docker/Podman is running, then type `$ yarn start`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
-   - Make sure Docker/Podman is running, then type `$ yarn start:stage`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
+   - Make sure podman desktop, or Docker, is running, then type `$ yarn start`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
+   - Make sure podman desktop, or Docker, is running, then type `$ yarn start:stage`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
```

Comment:
```suggestion
   - Make sure podman desktop is running, then type `$ yarn start`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
   - Make sure podman desktop is running, then type `$ yarn start:stage`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
```

No support anywhere for Docker Desktop. 🚫

Reply:
see #345 for resolve